content/en/integrations/guide/azure-portal.md

```diff
@@ -91,14 +92,24 @@ Select **Change Plan** on the overview page to change your Datadog billing plan.
 The portal retrieves all the available Datadog plans for your tenant, including any private offers. Select the appropriate plan and click **Change Plan**.
 
 ## Datadog org configurations
+
+### Monitored subscriptions
+
+Select **Monitored Subscriptions** in the left sidebar to view or modify the scope of the Datadog resource. A list of currently monitored subscriptions appears. Use this view to configure the scope of the Datadog resource to monitor as many subscriptions as desired.  The subscription with the Datadog resource must be included in the scope.
+
+   - To add subscriptions to monitor, click `+ Add Subscriptions`. The list of subscriptions available only includes subscriptions to which you have `Owner` role assigned. Select the subscriptions you wish to monitor and click `Add`. 
+   - To remove subscriptions from being monitored with Datadog, select the subscriptions you wish to remove and click `Remove Subscriptions`. You can only remove subscriptions to which you have `Owner` role assigned.
```

Comment:
```suggestion
   - To remove subscriptions from being monitored with Datadog, select the subscriptions you wish to remove and click `Remove Subscriptions`. You can only remove subscriptions to which you have the `Owner` role assigned.
```
Small suggestion. I generally avoid "of which / to which" constructions. They're a little formal. Perhaps: "Only accounts with the `Owner` role can remove subscriptions."